// vuepress-plugin-md-enhance/lib/node/i18n.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.i18n = void 0;
exports.i18n = {
    info: {
        "/zh/": "相关信息",
        "/en/": "Info",
        "/de/": "Information",
        "/vi/": "Thông tin",
        "/uk/": "Інформація",
        "/ru/": "Информация",
    },
    tip: {
        "/zh/": "提示",
        "/en/": "Tips",
        "/de/": "Tips",
        "/vi/": "Tips",
        "/uk/": "Поради",
        "/ru/": "Подсказки",
    },
    warning: {
        "/zh/": "注意",
        "/en/": "Note",
        "/de/": "Notiz",
        "/vi/": "Lưu ý",
        "/uk/": "Примітка",
        "/ru/": "Примечание",
    },
    danger: {
        "/zh/": "警告",
        "/en/": "Warning",
        "/de/": "Warnung",
        "/vi/": "Cẩn thận",
        "/uk/": "Увага",
        "/ru/": "Предупреждение",
    },
    detail: {
        "/zh/": "详情",
        "/en/": "Detail",
        "/de/": "Details",
        "/vi/": "Chi tiết",
        "/uk/": "Деталь",
        "/ru/": "Деталь",
    },
};
